client/my-sites/post-type-list/post-actions-ellipsis-menu/duplicate.jsx

```diff
@@ -16,23 +16,23 @@ import PopoverMenuItem from 'components/popover/menu-item';
 import { getPost } from 'state/posts/selectors';
 import canCurrentUserEditPost from 'state/selectors/can-current-user-edit-post';
 import { getEditorDuplicatePostPath } from 'state/ui/editor/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+import isJetpackModuleActive from 'state/selectors/is-jetpack-module-active';
 import { bumpStat, recordTracksEvent } from 'state/analytics/actions';
 import { bumpStatGenerator } from './utils';
-import { getSelectedEditor } from 'state/selectors/get-selected-editor';
-import isCalypsoifyGutenbergEnabled from 'state/selectors/is-calypsoify-gutenberg-enabled';
 
 function PostActionsEllipsisMenuDuplicate( {
 	translate,
 	canEdit,
 	status,
 	type,
+	copyPostIsActive,
 	duplicateUrl,
 	onDuplicateClick,
-	calypsoifyGutenberg,
 } ) {
 	const validStatus = includes( [ 'draft', 'future', 'pending', 'private', 'publish' ], status );
 
-	if ( ! canEdit || ! validStatus || 'post' !== type || calypsoifyGutenberg ) {
+	if ( ! canEdit || ! validStatus || 'post' !== type || ! copyPostIsActive ) {
```

Comment:
@kwight, if the component uses a selector that needs to be populated, we'd expect that component to also add the QueryComponent in the render function. There are some exceptions (say when we add query components to layout, but this help keeps it clear to the reader where the data is being populated from.)

In theory our data-layer should batch duplicate GET requests.

For QueryComponents to work, we always need to call this, so for example we may update this to:
```jsx
if ( shouldNotRenderUI ) {
    return ( <MyQueryComponent /> );
}

return (
<PopoverMenuItem>
    <MyQueryComponent />
    { myOtherChildren }
</PopOverMenuItem>
);
```

Reply:
Worked a treat. The menu item still suddenly pops in the first time, but it's fine, I think. Thanks @gwwar !